source/usage-examples/changeStream.txt

```diff
@@ -0,0 +1,65 @@
+=================
+Watch for Changes
+=================
+
+.. default-domain:: mongodb
+
+Open a Change Stream
+--------------------
+
+You can keep track of changes to data in MongoDB, such as changes to a
+collection, database, or deployment, by opening a :manual:`Change
+Stream</changeStreams/>`. A ``Change Stream`` allows applications to
+watch for changes to data and react to those changes. You can open a
+``Change Stream`` by calling the :node-api:`watch()
+</Collection.html#watch>` method on a collection or the equivalent
+``watch()`` method on a database, or deployment. 
+
+The ``watch()`` method takes an optional pipeline as the first parameter
```

Comment:
Optional: I think "pipeline" and "array of stages" might be too far apart in this sentence, and "change events output" is a little ambiguous. Consider:

> The ``watch()`` method optionally takes a pipeline, an array of stages, as the first parameter to filter the change events.

Reply:
Fixed almost verbatim to: ```The watch() method optionally takes a pipeline, an array of stages, as the first parameter to filter the `change events</link>`_ output.```